README.md

```diff
@@ -60,7 +60,7 @@ You can find samples demonstrating how to use ModelContextProtocol with an LLM S
 
 Clients can connect to any MCP server, not just ones created using this library. The protocol is designed to be server-agnostic, so you can use this library to connect to any compliant server.
 
-Tools can be exposed easily as `AIFunction` instances so that they are immediately usable with `IChatClient`s.
+Tools can be exposed easily as `McpClientTool` instances so that they are immediately usable with `IChatClient`s.
```

Comment:
Thanks for your contribution, but this isn't a typo. AIFunction isn't just another name for McpClientTool. McpClientTool inherits from AIFunction which is helpful because Microsoft.Extensions.AI and IChatClient doesn't have a dependency the ModelContextProtocol package or the McpClientTool type.

https://learn.microsoft.com/en-us/dotnet/core/extensions/artificial-intelligence?tabs=dotnet-cli#tool-calling

I think we can clarify this a bit though.

```suggestion
Tools can be easily exposed for immediate use by `IChatClient`s, because `McpClientTool` inherits from `AIFunction`.
```